SDWebImage/SDImageCache.m

```diff
@@ -59,7 +59,11 @@ - (nonnull instancetype)initWithNamespace:(nonnull NSString *)ns
                        diskCacheDirectory:(nonnull NSString *)directory
                                    config:(nullable SDImageCacheConfig *)config {
     if ((self = [super init])) {
-        NSString *fullNamespace = [@"com.hackemist.SDImageCache." stringByAppendingString:ns];
+        NSString *namespacePrefix = config.namespacePrefix;
+        if (!namespacePrefix) {
+            namespacePrefix = @"";
```

Comment:
we should make the default `"com.hackemist.SDImageCache."`

Reply:
`com.hackemist.SDImageCache.' it's already be the default value. See the description above and that `SDImageCacheConfig.m` files. This can become `nil` only when you explicitly set it with a `nil`. So I think maybe we do not need to protect more here. Just let it before the empty string `@""`